### PR TITLE
fix: Add command line arguments parameterization to Helm chart

### PR DESCRIPTION
**Pull request by VictorBac**

### CODE REVIEW

**Comment by grzesuav** [comment]
hi @VictorBac thanks for the PR, could hou however change the commit title to `fix: Add command line arguments parameterization to Helm chart`  to make it more obvius what it is about ?

**Review by grzesuav — CHANGES_REQUESTED** [review]
1. Please adjust commit message
2. Please add parameter descriptions to https://metacontroller.github.io/metacontroller/guide/helm-install.html in https://github.com/metacontroller/metacontroller/blob/master/docs/src/guide/helm-install.md

**Comment by VictorBac** [comment]
Done, but I am not sure about the default value for zap.devel:
https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/

I let it empty, if someone needs it they will know how to use it ^^'

File: deploy/helm/metacontroller/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: v2.0.10
```
Comment (grzesuav) [inline deploy/helm/metacontroller/Chart.yaml:2]: do not need to update this, the release automation will bump it, please restore it as it was

File: docs/src/guide/helm-install.md [hunk 2]
```diff
@@ -38,3 +38,9 @@ helm install metacontroller deploy/helm/metacontroller-v*.tgz
 | `nodeSelector`                            | Node labels for pod assignment                | `{}`                                                    |
 | `tolerations`                             | Toleration labels for pod assignment          | `[]`                                                    |
 | `affinity`                                | Affinity settings for pod assignment          | `{}`                                                    |
+| `zap.logLevel`                            | Zap Level to configure the verbosity of logging | `4`                                                   |
+| `zap.devel`                               | Development Mode or Production Mode           | `""`                                                    |
```
Comment (grzesuav) [inline docs/src/guide/helm-install.md:42]: default should be producion, so logging in json etc

**Comment by VictorBac** [comment]
Thank you @grzesuav , I fixed

**Review by grzesuav — APPROVED** [review]

**Pull request merged**